datadotworld/files.py

```diff
@@ -171,7 +233,15 @@ class RemoteFileException(Exception):
     """
     Exception wrapper for exceptions arising from the RemoteFile
     """
-
     def __init__(self, *args, **kwargs):
         self.cause = kwargs.pop('cause', None)
         super(RemoteFileException, self).__init__(*args, **kwargs)
+
+
+class NoSuchRemoteFileException(RemoteFileException):
```

Comment:
Consider renaming exception classes using `Error` as the suffix, instead of `Exception`.